windows/d2-installer.nsi

```diff
@@ -355,24 +375,49 @@ Function DetectVSAndSDK
     StrCpy $VCPath $0
     StrCpy $VCVer $1
 
-    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot81"
+	!insertmacro _DetectSDK "Windows Kits\Installed Roots" "KitsRoot10" "lib\um\x64"
     IfErrors 0 done_sdk
-    ClearErrors
-    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot" ; 8.0
+    !insertmacro _DetectSDK "Microsoft SDKs\Windows\v10.0" "InstallationFolder" "lib\um\x64"
```

Comment:
I was just extrapolating from the older versions here. I guess it can be removed.